src/Databases/MySQL/MaterializedMySQLSyncThread.cpp

```diff
@@ -853,7 +853,7 @@ void MaterializedMySQLSyncThread::onEvent(Buffers & buffers, const BinlogEventPt
                 return buf.str();
             };
 
-            LOG_DEBUG(log, "Skip MySQL event: \n {}", dump_event_message());
+            LOG_DEBUG(log, "Skip {} event: \n {}", to_string(receive_event->type()), dump_event_message());
```

Comment:
`magic_enum::enum_name(type)` is also an option instead of `to_string`

Reply:
hm, it looks even better, if `magic_enum::enum_name(type)` does the same, so it would be nice to get rid of all `to_string` here

Reply:
yes, it should be the same

Reply:
It is ok to make the changes in this PR.